### ISSUE TITLE
Signup Wizard: Partition not showing up on sign up page

### ISSUE THREAD

**Issue by marcierempel** [secc/RockPlugins#42]
I set up a Session partition (schedules) and it's not showing on the 'volunteer sign up form - Connections' block. The key for the Group Member attribute is 'Session' and I've set that as the Parameter Name as well as the Group Member Attribute Keys - URL in the volunteer sign up form block.
I've set the group member attribute Field Type to 'Schedules' instead of 'Schedule' so they can choose more than one Session. Do you know if this is a bug? The schedule selected is not coming through to the Connection either. It worked for us when we used the plugin at Christmas time. We're currently running Rock v8.6 and the latest version of the plugin.

**Comment by marcierempel** [comment]
I also noticed that the Placement group isn't coming through to the connection and the total numbers aren't updating in either the Volunteer Sign Up Wizard blocks.

**Comment by marcierempel** [comment]
Ok after a but of playing... I figured it out. The role in the wizard block needs to be mapped to the group for everything to show up right. I don't believe the documentation indicates that specifically.

**Comment by cbump** [comment]
Is everything working as desired now?  I'm hoping to release a new version this week which has a few additional features and fixes some bugs with counts and a few edge-case situations that are similar to what you described.

**Comment by marcierempel** [comment]
It looks like it. I think I just missed the step to map the groups in the block. If that step does not exist in the documentation, maybe update that?

Marcie Rempel | EXECUTIVE ASSISTANT
Sherwood Park Alliance Church | spac.ca<http://www.spac.ca/>
780.467.8404 x420

**Comment by cbump** [comment]
Hi Marcie,

A new version of the Signup Wizard plugin was released over the weekend.  I made a change to the plugin to fix an issue where it was requiring the placement group to be set when it was previously set on upstream partitions and shouldn't have needed to be set again.  I'm hoping this change addresses the issue you were seeing here. 

The documentation shows the Group being set on the Campus but not on the Role which, I believe, should work properly now.

Thanks!

Chuck Bump
Southeast Christian Church
